README.md

```diff
@@ -36,7 +43,11 @@ in WeeChat.
 If python-potr fails to install, you are probably missing some packages.
 To install all the requirements on a Debian/Ubuntu system, run
 
-    sudo apt-get install python-pip build-essential python-dev
+    sudo apt-get install python-pip python-wheel build-essential python-dev
+
+Or on Arch run
+
+    sudo pacman --needed -S python2-pip python2-wheel python2-keyring base-devel
```

Comment:
* `--needed` means "don't (re)install already installed packages".
* Pacman says that python2-keyring is optional depedency for python2-wheel for verifying wheel signatures.
    * verifying signatures of encryption related things just generally seems like a good idea.
* `base-devel` is Arch's "`build-essential`"